web/src/main/java/com/microsoft/applicationinsights/web/internal/WebRequestTrackingFilter.java

```diff
@@ -92,6 +97,13 @@ private void onException(Exception e) {
      */
     public void init(FilterConfig config){
         try {
+            if (!ClassDataUtils.INSTANCE.isMethodExists(javax.servlet.http.HttpServletResponse.class, "getStatus")) {
+                InternalLogger.INSTANCE.error(
+                        "Servlet Jars mismatch. The Java SDK needs the Servlet version 3.0, Web request tracking filter will be disabled.");
```

Comment:
Please change to the following:
_Unsupported servlet version. The Application Insights web request tracking filter requires the provided javax.servlet library to implement servlet spec 3.0 or above. The web request tracking filter will be disabled._